### ISSUE TITLE
front-end.css not loaded in public site

### ISSUE THREAD

**Issue by bugsysop** [studiopress/genesis-sample#141]
If I change this declaration in `functions.php` of my child theme:

````
// Defines the child theme (do not remove).
define( 'CHILD_THEME_NAME', 'Genesis Sample' );
define( 'CHILD_THEME_URL', 'https://www.studiopress.com/' );
define( 'CHILD_THEME_VERSION', '2.7.0' );
````
to

`````
// Defines the child theme (do not remove).
define( 'CHILD_THEME_NAME', 'My Theme' );
define( 'CHILD_THEME_URL', 'https://www.studiopress.com/' );
define( 'CHILD_THEME_VERSION', '1.0' );
`````

The `front-end.css` file for Gutenberg is not loaded anymore in site header

**Comment by nickcernis** [comment]
Thank you for this report, @bugsysop.

This is because the front-end.css file depends on the genesis-sample stylesheet by name here:

https://github.com/studiopress/genesis-sample/blob/6801a3f502abb9d38b88f671e668e4e75004fdac/lib/gutenberg/init.php#L19-L24

If you rename your theme, it's best to also rename all `genesis_sample` and `genesis-sample` strings to match your new theme name in the code (to `my_theme` and `my-theme`), which will prevent issues like this one.

I'll leave this open to look into ways to prevent the need for this, though. We could either:

- Create an npm script or mini site to make theme renaming easier.
- Pull the child theme name from the constant automatically and use that in the dependency, instead of hard-coding `genesis-sample`.

**Comment by nathanrice** [comment]
>Resolved in 69c3f36. The CHILD_THEME_NAME will now be used for the dependency automatically.

💯
